Dockerfile

```diff
@@ -20,7 +20,7 @@ COPY . ./
 
 ENV APP_HOST=0.0.0.0
 ENV APP_PORT=80
-ENV RELOAD=""
+ENV RELOAD="true"
```

Comment:
Do you need to merge latest master? This shouldn't be a diff https://github.com/arturo-ai/arturo-stac-api/blob/master/Dockerfile#L23

Reply:
yeah I did after I started this PR 🤦